app/src/test/java/uk/ac/ebi/atlas/experimentpage/TSnePlotSettingsServiceTest.java

```diff
@@ -214,6 +218,17 @@ void noPerplexities() {
         assertThat(subject.getAvailablePerplexities(EXPERIMENT_ACCESSION)).isEmpty();
     }
 
+    @Test
+    @DisplayName("Valid cell count from database")
+    void validCellCount() {
+        Integer expectedCellCount = 100;
+        when(experimentCellCountDaoMock.fetchNumberOfCellsByExperimentAccession(EXPERIMENT_ACCESSION))
```

Comment:
You can use hardcoded `100` instead of an extra local variable as we are using it in one place.

Reply:
We are using `expectedCellCount` at two places in this test case, and if we hard code `100` then it will need to be done at two places. So imo that would affect the maintainability of the code in the future if we need to change this value then it would need to be changed in two places rather than in just one place.

Reply:
yeah, agree with you :).